### PR TITLE
[eclipse/xtext#1249] use bom for version management

### PR DESCRIPTION
**Pull request by szarnekow**

### CODE REVIEW

File: gradle/versions.gradle [hunk 1]
```diff
@@ -9,18 +9,19 @@ ext.versions = [
...
 	'gradle_plugins': '0.1.0',
 	'xtext_gradle_plugin': '2.0.2',
-	'equinoxCommon' : '3.9.0',
-	'eclipseCore': '3.13.0',
-	'emfEcore': '2.12.0',
-	'emfCodegen': '2.11.0',
-	'emfMwe': '1.3.21.201705291010',
-	'emfMwe2': '2.9.1.201705291010',
+	'dependency_management_plugin' : '1.0.6.RELEASE',
+	//'equinoxCommon' : '3.9.0',
```
Comment (szarnekow) [inline gradle/versions.gradle]: Remove comments

File: gradle/bootstrap-setup.gradle [hunk 2]
```diff
@@ -12,6 +12,9 @@ repositories {
 		name 'xtend-bootstrap'
 		url "$JENKINS_URL/job/xtend-bootstrap/lastStableBuild/artifact/build-result/maven-repository/"
 	}
+	// maven {
```
Comment (szarnekow) [inline gradle/bootstrap-setup.gradle]: Remove

File: gradle/versions.gradle [hunk 3]
```diff
@@ -9,18 +9,19 @@ ext.versions = [
...
+	//'eclipseCore': '3.13.0',
+	//'emfEcore': '[2.10.2,3)',
+	//'emfMwe': '1.3.21.201705291010',
+	//'emfMwe2': '2.9.1.201705291010',
+	// TODO why is this not commented out
 	'jdtCore': '3.13.102',
```
Comment (cdietrich) [inline gradle/versions.gradle]: is this still used?

File: org.eclipse.xtext.common.types.tests/build.gradle [hunk 4]
```diff
@@ -4,23 +4,14 @@ apply from: "${rootDir}/gradle/mwe2-workflows.gradle"
...
-	compile("org.eclipse.jdt:org.eclipse.jdt.compiler.tool:$versions.jdtCompilerTool") {
-		exclude group: "org.eclipse.jdt", module: "org.eclipse.jdt.core"
-	}
+	compile "org.eclipse.platform:org.eclipse.equinox.common"
+	compile "org.eclipse.platform:org.eclipse.core.runtime"
+	compile "org.eclipse.jdt:org.eclipse.jdt.core"
+	compile "org.eclipse.jdt:org.eclipse.jdt.compiler.apt"
+	compile "org.eclipse.jdt:org.eclipse.jdt.compiler.tool"
 	compile "org.eclipse.xtext:org.eclipse.xtext.testing:$versions.xtext"
 	compile "com.google.errorprone:error_prone_annotations:2.0.15"
```
Comment (cdietrich) [inline org.eclipse.xtext.common.types.tests/build.gradle]: this version is not managed by bom?

File: org.eclipse.xtext.java.tests/build.gradle [hunk 5]
```diff
@@ -2,7 +2,7 @@ dependencies {
 	compile project(':org.eclipse.xtext.java')
 	compile project(':org.eclipse.xtext.xbase.testdata')
 	compile "org.eclipse.xtext:org.eclipse.xtext.testing:$versions.xtext"
-	compile "junit:junit:$versions.junit"
+	compile "junit:junit"
 	compile "com.google.errorprone:error_prone_annotations:2.0.15"
```
Comment (cdietrich) [inline org.eclipse.xtext.java.tests/build.gradle]: is this version still managed here and not via bom?

File: org.eclipse.xtext.common.types.tests/build.gradle (hunk 4 above)
Comment (szarnekow) [reply org.eclipse.xtext.common.types.tests/build.gradle]: Done

**Comment by szarnekow** [comment]
Ready to be merged

**Pull request merged**